Doc/library/platform.rst

```diff
@@ -145,8 +145,9 @@ Cross Platform
 
 .. function:: system()
 
-   Returns the system/OS name, e.g. ``'Linux'``, ``'Windows'``, or ``'Java'``. An
-   empty string is returned if the value cannot be determined.
+   Returns the system/OS name, e.g. ``'Linux'``, ``'Windows'``, ``'Darwin'``
```

Comment:
If we're going to reword this, let's also try to not imply that this is an exhaustive list.  It doesn't include the BSD's, for example.  How about something like:
`Returns the system/OS name, among others ``'Linux'``, ...`

Reply:
I used "such as", hope that's equally fine. Is there an exhaustive list somewhere that we could point to?